packages/mermaid/src/diagrams/mindmap/mindmapRenderer.ts

```diff
@@ -155,38 +158,33 @@ function positionNodes(cy) {
   });
 }
 
-/**
- * Draws a an info picture in the tag with id: id based on the graph definition in text.
- *
- * @param {any} text
- * @param {any} id
- * @param {any} version
- * @param diagObj
- */
-
-export const draw = async (text, id, version, diagObj) => {
+export const draw = async (text: string, id: string, version: string, diagObj: Diagram) => {
   const conf = getConfig();
-
+  const db = diagObj.db as MindmapDB;
   conf.htmlLabels = false;
 
   log.debug('Rendering mindmap diagram\n' + text, diagObj.parser);
 
-  const securityLevel = getConfig().securityLevel;
+  const securityLevel = conf.securityLevel;
   // Handle root and Document for when rendering in sandbox mode
-  let sandboxElement;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let sandboxElement: any;
   if (securityLevel === 'sandbox') {
     sandboxElement = select('#i' + id);
   }
-  const root =
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const root: any =
     securityLevel === 'sandbox'
       ? select(sandboxElement.nodes()[0].contentDocument.body)
       : select('body');
-  // Parse the graph definition
 
   const svg = root.select('#' + id);
 
   svg.append('g');
```

Comment:
I'm not sure, but shouldn't use the `Group` element itself instead of the `SVG` element:

```typescript
import type { DrawDefinition, Group, SVG } from '../../diagram-api/types.js';
```

```suggestion
  const group: Group = svg.append('g');
```

Reply:
I've removed that svg.append. The other g is coming from `appendDivSvgG`, removing without would be out of scope for this PR. 
I don't think there is a need for a parent

Existing
<img width="251" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/931f7822-0e9c-4975-8779-be2fc1d52fd1">

Updated
<img width="262" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/dcc7b626-f106-4a78-a23c-e7fd16053878">